src/Core/Auth/Models/Mail/RegisterVerifyEmail.cs

```diff
@@ -4,9 +4,11 @@ namespace Bit.Core.Auth.Models.Mail;
 
 public class RegisterVerifyEmail : BaseMailModel
 {
-    public string Url => string.Format("{0}/finish-signup?token={1}",
+    public string Url => string.Format("{0}/finish-signup?token={1}&email={2}",
```

Comment:
Adding email to URL so that the registration finish component can have access to email in order to create a master key once the user enters their MP.